README.md

```diff
@@ -39,15 +39,15 @@ A youtube gameplay video can be found here: https://www.youtube.com/watch?v=-PEK
 * Changes in UI
   * We have added an instructions scene and a stunning menu with automatic gameplay on the background
 * Changes in logic
-  * We have fixed uncountable bugs which made the game unplayable
+  * We have fixed a very countable number of bugs
 
 <br></br>
-In short, we have turned a quite buggy game into a good looking, bug-free and addictive piece of art.
+In short, we have turned a quite buggy game into a good looking, bug-free and addictive piece of art while destroying the core game design that made Chromacore interesting.  The original authors released an updated version post-GitHub Game Off, [here](https://github.com/Murkantilism/Chromacore), preserving what made the game fun to begin with.
```

Comment:
Then in that case it wasn't our game that was unplayable, it was incompatibility of software on your end that required you to update some code so that your software (Unity version 4.x) could compile it.

You do realize how those two are different, right? The first implies we submitted/released an unplayable game for GitHub GO 2013, the second is a pretty routine thing that happens when you try to use software that was compiled with a different version than what you have installed.